legnardotmnt.yml

```diff
@@ -0,0 +1,27 @@
+class: leonardotmnt
+costumes:
+- chatcolor: '&1'
+  name: &2Leo&1nar&2do&8(&2TMNT&8)
```

Comment:
Change the '&' symbols to the § symbols, you can copy from this message and just replace them.

Reply:
confirmed correct